operator/option/config.go

```diff
@@ -209,6 +209,12 @@ const (
 
 	// AWSPaginationEnabled toggles pagination for AWS EC2 API requests
 	AWSPaginationEnabled = "aws-pagination-enabled"
+
+	// AWSEnableRouteTableDiscovery enables route table discovery and route-table-aware subnet selection.
+	// Disabling this option can significantly reduce memory usage and AWS API calls in environments with
+	// many route tables, or where route-table-aware subnet selection is not needed (e.g., when pod subnets
+	// are in different subnets from node subnets).
```

Comment:
(e.g., when pod subnets
	// are in different subnets from node subnets)

I don't think this is accurate since pods subnets can be different from nodes subnets. Its more about pods has its own route table which is different from node